### PR TITLE
Script to scale and Upgrade

### PR DESCRIPTION
**Pull request by Bhuvaneswari-Santharam**
This script is wrapper to do the aks engine scale and upgrade operations

### CODE REVIEW

File: operations/aksEngine/aksEngineScale.sh [hunk 1]
```diff
@@ -0,0 +1,193 @@
+#! /bin/bash
+set -e
+
+log_level()
```
Comment (rjaini) [inline operations/aksEngine/aksEngineScale.sh]: log_level()
{
    case "$1" in
        -e) echo "$(date) [Err]  " ${@:2}
        ;;
        -w) echo "$(date) [Warn] " ${@:2}
        ;;
        -i) echo "$(date) [Info] " ${@:2}
        ;;
        *)  echo "$(date) [Debug] " ${@:2}
        ;;
    esac
}

File: operations/aksEngine/aksEngineScale.sh [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+
+
+
+# Validate input
+
+if [ -z "$TENANT_ID" ]
+
+then
+
+    echo ""
```
Comment (rjaini) [inline operations/aksEngine/aksEngineScale.sh]: Replace all echo statement to log-level function.

File: operations/aksEngine/runAksScript.sh [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+scp -i $IDENTITYFILE $IDENTITYFILE $USER@$DVM_HOST:/home/$AZURE_USER/.ssh/id_rsa
+
+
+ROOT_PATH=/home/azureuser
+FILENAME=$(basename $FILE)
+download_scripts $FILE $FILENAME
+
+scp -q -i $IDENTITYFILE $SCRIPTSFOLDER/*.sh $USER@$DVM_HOST:$ROOT_PATH
+
+if [ $FILENAME == "aksEngineScale.sh" ] ; then
```
Comment (rjaini) [inline operations/aksEngine/runAksScript.sh]: good to have an option rather depend on filename.

File: operations/aksEngine/aksEngineScale.sh (hunk 1 above)
Comment (Bhuvaneswari-Santharam) [reply operations/aksEngine/aksEngineScale.sh]: What difference does it make to have it the way you suggested?

Comment (rjaini) [reply operations/aksEngine/aksEngineScale.sh]: To be consistent with one flow above is the format typically followed in Linux.

**Pull request merged**